easybuild/easyblocks/b/bamtools.py

```diff
@@ -87,7 +87,8 @@ def sanity_check_step(self):
                                           'lib/libjsoncpp.a'])
             custom_paths['dirs'].extend(['include/api', 'docs'])
         else:
-            custom_paths['files'].extend(['include/bamtools/shared/bamtools_global.h', 'lib64/libbamtools.a'])
-            custom_paths['dirs'].extend(['include/bamtools/api', 'lib64/pkgconfig'])
+            custom_paths['files'].extend(['include/bamtools/shared/bamtools_global.h',
+                                          ('lib/libbamtools.a', 'lib64/libbamtools.a')])
+            custom_paths['dirs'].extend(['include/bamtools/api', ('lib.pkgconfig', 'lib64/pkgconfig')])
```

Comment:
@boegel Here is a typo. It should be `lib/pkgconfig` instead of `lib.pkgconfig`.